pom.xml

```diff
@@ -1,9 +1,13 @@
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+  <properties>
+    <jenkins.version>2.120</jenkins.version>
```

Comment:
Better to use an LTS version as a baseline. Also, this section usually goes after `<packaging>`

Reply:
2.249.1 should suffice?

Reply:
This build will fail if you don't include `<java.level>8</java.level>` in the `<properties>` block too. The only reason I am suggesting 8 over something newer is because this plugin is pretty old.